lmd/objects.go

```diff
@@ -275,7 +275,7 @@ func NewHostsTable() (t *Table) {
 	t.AddColumn("custom_variable_names", Static, StringListCol, "A list of the names of all custom variables")
 	t.AddColumn("custom_variable_values", Dynamic, StringListCol, "A list of the values of the custom variables")
 	t.AddColumn("display_name", Static, StringCol, "Optional display name of the host")
-	t.AddColumn("event_handler", Static, StringCol, "Naemon command used as event handler")
```

Comment:
could you please move this to the end of this function to the other optional columns.

Reply:
Done